crates/fluvio-controlplane-metadata/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-controlplane-metadata"
 edition = "2021"
-version = "0.19.0"
+version = "0.20.0"
```

Comment:
I don't think we need to bump a major bump since we didn't actually change API.   so only patch version increase i s needed

Reply:
Okay! I thought given that we introduced a different struct this represented a change in API. But as we plan to keep it private and for internal usage makes sense to me as well. Thanks!